addons/website_portal/controllers/main.py

```diff
@@ -9,6 +9,16 @@
 from odoo.fields import Date
 
 
+def get_records_pager(ids, current):
```

Comment:
I would put it into website with other helper methods. What is current ? It seems it is a record set. 

By the way you should maybe choice between ids and records. It seems you populate the pager with record_set.ids, and use a record as an entry point. Why don't you directly use a record set instead of ids ? With maybe a mean to clean it regularly.